xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -344,6 +344,9 @@ img=Attached Image
 imgtext=example.jpg
 sign=Signature
 
+### Skip to
+skipTo.content=Skip to Content
```

Comment:
The key doesn't really follow the convention: https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/
Generally speaking the translation key should help identifying where the it's used, which is clearly not the case here.

Reply:
Thank you surli for the review and for the link to L10N conventions, I tried to propose a translation key that is more aligned with the rules, please let me know if you have better suggestions.